<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>795</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">

*{
font-family: century gothic;
}
QWidget{
background-color: rgb(26,28,33);
}
QGroupBox{
border:2px  solid rgb(45, 137, 239);
color:white;
background-color: rgb(39,44,54);
border:None;
}
QPushButton{
background-color: rgb(103, 103, 103);
border-radius:0px;
color:white;
font-weight:bold;
}

QPushButton:hover{
border:2px solid rgba(45, 137, 239,100);
border-radius:0px;
color:white;
font-weight:bold;
}







</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">

*{
font-family: century gothic;
}

QPushButton{
background-color: qlineargradient(spread:pad, x1:0.482, y1:0.3, x2:0.482224, y2:0, stop:0.15 rgba(0, 0, 0, 147), stop:1 rgba(255, 255, 255, 25));
/*background:#2d89ef;*/
border-radius:0px;
color:white;
font-weight:bold;
}

QPushButton:hover{
border:1px solid rgba(45, 137, 239,100);
border-radius:0px;
color:white;
font-weight:bold;
}








</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>7</number>
        </property>
        <property name="topMargin">
         <number>7</number>
        </property>
        <property name="rightMargin">
         <number>7</number>
        </property>
        <property name="bottomMargin">
         <number>7</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>century gothic</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background:none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../graphics/resource.qrc">:/icon/icon/dome_camera_80px.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>century gothic</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:white;
background:none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="text">
           <string>CCTV Based Restricted Area Monitoring System</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="topMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>100</number>
      </property>
      <property name="spacing">
       <number>50</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_live_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Nirmala UI</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-icon: url(:/icon/icon/record_96px.png);
font: 12pt &quot;Nirmala UI&quot;;</string>
        </property>
        <property name="text">
         <string>Live View</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_device_manager">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-icon: url(:/icon/icon/bullet_camera_96px.png);
font: 12pt &quot;Nirmala UI&quot;;</string>
        </property>
        <property name="text">
         <string>Device Manager</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnFace">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-icon: url(:/icon/icon/dome_camera_80px.png);
font: 12pt &quot;Nirmala UI&quot;;</string>
        </property>
        <property name="text">
         <string>Face Records</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btn_about">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-icon: url(:/icon/icon/about_96px.png);
font: 12pt &quot;Nirmala UI&quot;;</string>
        </property>
        <property name="text">
         <string>About</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_settings">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-icon: url(:/icon/icon/motion_detector_64px.png);
font: 12pt &quot;Nirmala UI&quot;;</string>
        </property>
        <property name="text">
         <string>Add Person</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btn_logs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>220</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">qproperty-icon: url(:/icon/icon/checked_user_male_96px.png);
font: 12pt &quot;Nirmala UI&quot;;</string>
        </property>
        <property name="text">
         <string>Attendance Logs</string>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>century gothic</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
background-color: rgb(42, 54, 59);</string>
     </property>
     <property name="text">
      <string>Copyright © 2024 , Group 04. All rights reserved.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../graphics/resource.qrc"/>
 </resources>
 <connections/>
</ui>
